From: Aurélien Gâteau <agateau@kde.org>
Date: Tue, 01 May 2012 17:20:01 +0000
Subject: Unbreak binary compatibility
X-Git-Url: http://quickgit.kde.org/?p=kdelibs.git&amp;a=commitdiff&amp;h=dc217720bd685a30ef7914f09ab8c1e321b92c88
---
Unbreak binary compatibility

/me puts his head under a brown bag

CCMAIL: cgiboudeaux@gmx.com
---


--- a/kdeui/widgets/kmessagewidget.cpp
+++ b/kdeui/widgets/kmessagewidget.cpp
@@ -331,6 +331,13 @@ void KMessageWidget::paintEvent(QPaintEv
     }
 }
 
+void KMessageWidget::showEvent(QShowEvent* event)
+{
+    // Keep this method here to avoid breaking binary compatibility:
+    // QFrame::showEvent() used to be reimplemented.
+    QFrame::showEvent(event);
+}
+
 bool KMessageWidget::wordWrap() const
 {
     return d->wordWrap;

--- a/kdeui/widgets/kmessagewidget.h
+++ b/kdeui/widgets/kmessagewidget.h
@@ -161,6 +161,8 @@ protected:
 
     void resizeEvent(QResizeEvent *event);
 
+    void showEvent(QShowEvent *event);
+
 private:
     KMessageWidgetPrivate *const d;
     friend class KMessageWidgetPrivate;

